### PR TITLE
Galtoubul todo

### PR DESCRIPTION
**Pull request by galtoubul**

### CODE REVIEW

File: src/ex2/main.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  }
+
+  onTaskDelete(todoToDelete) {
+    document.querySelector(`#task-${todoToDelete.id}`).remove();
+    this.diplayedTasksSet.delete(todoToDelete.id);
+    this.updateTasksLeft();
+  }
+
+  onClearAll() {
+    const nodes = document.querySelector(newTasks).childNodes;
```
Comment (liorshamian) [inline src/ex2/main.js:97]: if you would render the todos from itemManager you could make your solution simpler (just assign empty array to todos and render it)

File: src/ex2/main.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    );
+    const tasksLeftElem = document.querySelector(".tasks-left");
+    tasksLeftElem.innerText = `Keep Grinding! You got ${tasksLeft} to go`;
+  }
+
+  createTask({ id: taskId, value }) {
+    const task = document.createElement("div");
+    task.classList.add("task-container");
+    task.id = `task-${taskId}`;
+    task.innerHTML = `
```
Comment (liorshamian) [inline src/ex2/main.js:119]: there's a better way to do it. it's not a good convention to add html code inside js

**Review by liorshamian — APPROVED** [review]
you could make your code shorter & simpler, please learn from our presented solution.
but all in all - good job!

File: src/ex2/main.js (hunk 2 above)
Comment (galtoubul) [reply src/ex2/main.js:119]: By using createElement?

**Pull request merged**